libgalois/src/GraphTopology.cpp

```diff
@@ -164,6 +165,11 @@ katana::EdgeShuffleTopology::Make(tsuba::RDGTopology* rdg_topo) {
       &(rdg_topo->edge_index_to_property_index_map()[rdg_topo->num_edges()]),
       edge_prop_indices.begin());
 
+  // Since we copy the data we need out of the RDGTopology into our own arrays,
+  // unbind the RDGTopologys file store to save memory.
+  auto res = rdg_topo->unbind_file_storage();
+  KATANA_LOG_ASSERT(res);
```

Comment:
why not `KATANA_CHECKED`?

Reply:
Currently, the whole stack of BlahTopology()::Make, PGViewCache::BuildOrGetEdgeShuffleTopo, PGViewCache:;BuildView, don't return katana::Result<>. Making this `KATANA_CHECKED` would require changing the signatures of a dozen functions, and require changes to the enterprise repo as well.
 
Since my focus here is on memory usage, I feel doing that work is out of scope.
Also, if we can't unbind the `file_store` something huge has gone wrong and we should probably die anyway. 

- Resolving the dittos for now

Reply:
Ah right, I forgot that the parent function **also** needs to return a `Result` type. I agree, that task is out-of-scope for now.